SPECS/fribidi/fribidi.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "fribidi-1.0.9.tar.xz": "c5e47ea9026fb60da1944da9888b4e0a18854a0e2410bbfe7ad90a054d36e0c7"
+  "fribidi-1.0.10.tar.xz": "7f1c687c7831499bcacae5e8675945a39bacbad16ecaa945e9454a32df653c01"
```

Comment:
I think you still need to upload the sources into the blob storage.

Reply:
I will be uploading the source tar.xz using the pipeline.